portfolio/src/main/java/com/google/sps/servlets/PokedexDataServlet.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/pokedex-data")
+public class PokedexDataServlet extends HttpServlet {
+
+  private int[] getGenerationCount() {
+    int[] result = new int[10];
```

Comment:
I don't know much about pokemon, but how was 10 chosen here?

perhaps if its meaningful, extract the "magic number" to a constant with an explicit name so that other contributors know why it's chosen.


static final int TOTAL_POKEMON_GENERATIONS or something